src/pages/workspace/reimburse/WorkspaceReimburseNoVBAView.js

```diff
@@ -93,17 +84,14 @@ class WorkspaceReimburseNoVBAView extends React.Component {
     }
```

Comment:
I feel like this could be just `numValue.toFixed(2);`. Right?

Reply:
Sorry, didn't understand

Reply:
I mean 
```js
const fraction = numValue.toString().split('.')[1];
        return !fraction || fraction.length < 2
            ? numValue.toFixed(2)
            : numValue.toString();
 ```
 
 could be just  numValue.toFixed(2);

Reply:
Done